### ISSUE TITLE
Another solve() problem

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#2373]
Imported from SourceForge on 2024-07-06 07:10:05
Created by **[matheschlumpf](https://sourceforge.net/u/matheschlumpf/)** on 2007-12-17 09:04:56
Original: https://sourceforge.net/p/maxima/bugs/1302

---

Maxima 5.13.0 http://maxima.sourceforge.net
Using Lisp GNU Common Lisp \(GCL\) GCL 2.6.8 \(aka GCL\)
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
This is a development version of Maxima. The function bug\_report\(\)
provides bug reporting information.
\(%i1\) solve\(\[x^2+y^2=2^2,\(x-1\)^2+\(y-1\)^2=2^2\]\);
\(%o1\)                                 \[\]
\(%i2\)

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 07:10:06
Created by **[crategus](https://sourceforge.net/u/crategus/)** on 2010-10-12 18:27:11
Original: https://sourceforge.net/p/maxima/bugs/1302/#1fa8

---

Copying the example from the bug report ID: 2037060 - Symbolic Center of Circle Calculation into this thread. It is the same problem:

\-------------------------------------------
\(%i1\) s: solve\(\[\(x-x0\)^2+\(y-y0\)^2=r^2,\(x-x1\)^2+\(y-y1\)^2=r^2\],\[x,y\]\);
\(%o1\) \[\]
\(%i2\)

Maxima uses up a lot of processing power and doesn't find the correct answer.

ftp://ftp.deltacompsys.com/public/NG/Mathcad%20-%20CenterOfCircleCalculations.pdf

Peter Nachtwey
peter@deltamotion.com
\-------------------------------------------

Dieter Kaiser

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 07:10:10
Created by **[billingd](https://sourceforge.net/u/billingd/)** on 2016-10-02 07:02:14
Original: https://sourceforge.net/p/maxima/bugs/1302/#67bb

---

- **status**: open --> closed
- **assigned_to**: David Billinghurst
- **Group**:  --> None

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 07:10:13
Created by **[billingd](https://sourceforge.net/u/billingd/)** on 2016-10-02 07:02:14
Original: https://sourceforge.net/p/maxima/bugs/1302/#1fef

---

Fixed by commit [2986fc]  See [bugs:#3208] for details

~~~
(%i1) solve([x^2+y^2=2^2,(x-1)^2+(y-1)^2=2^2]);

                                        sqrt(7)      1
                               sqrt(2) (------- - -------) + 2
              sqrt(7) - 1               sqrt(2)   sqrt(2)
(%o1) [[y = - -----------, x = -------------------------------],
                   2                          2
                                                       sqrt(7)      1
                                              sqrt(2) (------- + -------) - 2
                           sqrt(7) + 1                 sqrt(2)   sqrt(2)
                      [y = -----------, x = - -------------------------------]]
                                2                            2
(%i2) time(%o1);

(%o2)                               [0.016]
~~~
